### PR TITLE
Speed up boot time

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Also disabled KPTI and removed some virtual framebuffer, since its not needed

File: device.mk [hunk 1]
```diff
@@ -53,6 +53,18 @@ PRODUCT_COPY_FILES += \
     frameworks/av/services/audiopolicy/config/a2dp_in_audio_policy_configuration.xml:$(TARGET_COPY_OUT_VENDOR)/etc/a2dp_in_audio_policy_configuration.xml \
     frameworks/av/services/audiopolicy/config/bluetooth_audio_policy_configuration.xml:$(TARGET_COPY_OUT_VENDOR)/etc/bluetooth_audio_policy_configuration.xml
 
+PRODUCT_DEXPREOPT_SPEED_APPS += \
```
Comment (danielml3) [inline device.mk:57]: This should be moved along with the below settings into a #ART section. As a suggestion for having it tidy:

```
# ART
PRODUCT_DEXPREOPT_SPEED_APPS += \
    Settings \
    SystemUI

PRODUCT_MINIMIZE_JAVA_DEBUG_INFO := true
PRODUCT_ART_TARGET_INCLUDE_DEBUG_BUILD := false
PRODUCT_SYSTEM_SERVER_COMPILER_FILTER := speed-profile
PRODUCT_ALWAYS_PREOPT_EXTRACTED_APK := true
WITH_DEXPREOPT_DEBUG_INFO := false
USE_DEX2OAT_DEBUG := false
```

File: properties/vendor.prop [hunk 2]
```diff
@@ -61,6 +61,20 @@ vendor.audio_hal.period_size=192
 vendor.mm.enable.qcom_parser=16777215
 vendor.voice.path.for.pcm.voip=false
 
+### ART
```
Comment (danielml3) [inline properties/vendor.prop]: Use a single # instead of ###

**Review by danielml3 — COMMENTED** [review]
I left some suggestions for having more consistency.

**Comment by ghost** [comment]
Done

**Pull request merged**